src/component/CodeBlock.tsx

```diff
@@ -11,6 +11,11 @@ import json from "react-syntax-highlighter/dist/cjs/languages/hljs/json";
 import markdown from "react-syntax-highlighter/dist/cjs/languages/hljs/markdown";
 import shell from "react-syntax-highlighter/dist/cjs/languages/hljs/shell";
 import { useDarkMode } from "../hook/theme";
+import FileCopyIcon from '@material-ui/icons/FileCopy';
+import CheckIcon from '@material-ui/icons/Check';
+import { IconButton, Container, Zoom } from "@material-ui/core";
+import { makeStyles } from '@material-ui/core/styles';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
```

Comment:
unrelated changes. please merge master branch.

Reply:
@ry 
The branch was merged, the only commit is "Case insensitive filter in RegistryIndex" this shows all the history from the first fork.

Thansk